src/components/Navigator/Navigator.tsx

```diff
@@ -57,21 +62,21 @@ export function Navigator(props: INavigatorProps) {
         variant="icon"
         customClass="!border border-c-default disabled:border-tertiary"
         onClick={onClickBack}
-        disabled={activePage.index === 0}
+        disabled={activeItem === 0}
```

Comment:
It's a boolean, no!?

```suggestion
        disabled={activeItem}
```

Reply:
rename by `activeItemIndex`